src/libsync/account.cpp

```diff
@@ -160,6 +162,12 @@ void Account::setCredentials(AbstractCredentials *cred)
 
     _am = _credentials->createAM();
 
+    const QString cacheLocation = QStringLiteral("%1/accounts/%2/network/").arg(QStandardPaths::writableLocation(QStandardPaths::CacheLocation), _uuid.toString());
```

Comment:
Can you add an example.
Is the app name included?

Reply:
`~/.cache/ownCloud/accounts/<uuid>/network` on Linux.